batchflow/models/torch/base.py

```diff
@@ -789,6 +795,59 @@ def head(cls, inputs, **kwargs):
         """
         return cls.block(inputs, name='head', **kwargs)
 
+    # Model weights initialization
+    def init_weights(self, init_model_weights=None, init_zero_bias=False):
+        """ Initialize model weights with specific distribution and initialize biases to 0.
+
+        About weighs initialization, you can read here: `torch.nn.init <https://pytorch.org/docs/stable/nn.init.html>`_.
+
+        Parameters
+        ----------
+        init_model_weights : callable, 'common_used', or None
+            Model weights initilaization.
+            If None, then default initialization is used.
+            If 'common_used', then common used non-default initialization is used.
+            If callable, then callable applied to each layer.
+        init_zero_bias : bool
+            Whether to initialize all biases to zero.
+
+        Examples
+        --------
+            .. code-block:: python
+
+                def callable_init(module): # example of a callable for init
+                    if isinstance(module, nn.Linear):
+                        nn.kaiming_normal_(module.weight)
+
+                config = {'init_model_weights': callable_init}
+        """
+        def common_used_weights_init(module):
```

Comment:
Move it to a separate `initialization.py` file, add the source of this init function